Frends.AzureBlobStorage.DownloadBlob/Frends.AzureBlobStorage.DownloadBlob/DownloadBlob.cs

```diff
@@ -8,23 +8,23 @@
 using Azure.Storage.Blobs;
 
 #pragma warning disable CS1591
+#pragma warning disable CS1573
 
 namespace Frends.AzureBlobStorage.DownloadBlob
 {
     public class AzureBlobStorage
```

Comment:
Minor thing - maybe make this static, so it is clear that this is not instantiatable class?

Reply:
Changed to static.